File: TouchControl/src/io/github/JakeJMattson/touchcontrol/touchables/Pad.java
package io.github.JakeJMattson.touchcontrol.touchables;

import org.opencv.core.*;

/**
 * Abstract Pad - contains general pad behavior.
 *
 * @author JakeJMattson
 */
public abstract class Pad extends Touchable
{
	protected Pad(Rect dimensions, Scalar color)
	{
		super(dimensions, color);
	}

	/*
	 * (non-Javadoc)
	 * @see io.github.JakeJMattson.touchcontrol.touchables.Touchable#toString()
	 */
	@Override
	public String toString()
	{
		return super.toString();
	}
}

File: src/main/kotlin/io/github/jakejmattson/touchcontrol/touchables/Pad.kt
package io.github.jakejmattson.touchcontrol.touchables

import org.opencv.core.*

/**
 * Abstract Pad - contains general pad behavior.
 *
 * @author JakeJMattson
 */
abstract class Pad protected constructor(dimensions: Rect, color: Scalar): Touchable(dimensions, color)